IssueTracker/IssueTracker.FileSystem/Repository/BlobConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace IssueTracker.FileSystem;
+public class BlobConfiguration : IBlobConfigurationFactory
+{
+    public string Container { get; }
+    public string ConnectionString { get; }
+    public string AccountName { get; }
+    public string AccountKey { get; }
+    public BlobConfiguration(string container, string connstring, string accountName, string accountKey)
```

Comment:
Comply with the SA1201 ordering rules
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1201.md

Reply:
done